README.md

```diff
@@ -32,8 +40,7 @@ Here are the steps you'll take:
 You'll need to add a webhook that points to your newly built container image,
 either with the FQDN or the IP address and then add the `/api` Endpoint.
 Example: https://192.168.1.101:3000/api or
-https://bitbucket-quay-connector.cloudapps.master.ose/api After that be sure to
-add the string you set for the `SERVER_SECRET`.
+https://bitbucket-quay-connector.cloudapps.master.ose/api. The server secret does not need to be configured.
```

Comment:
Mentioning server secret will confuse people. We do not need to say:
> The server secret does not need to be configured.

Once this is merged, the `SERVER_SECRET` env variable is only available in git history.